department-of-veterans-affairs/GIBCT-Data-Service
# Update GIDS to accept new 109 School Closures File 19185

## Description

As a GIDS Tool user, I need the tool to import 109 School Closure data and populate data in the institutions.csv file, so the GI Bill Comparison Tool Data Dashboard file can display that information.

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19185

## Testing done

- Open GIDS 
- Navigate to the Sec109ClosedSchool and click upload
- Upload the sec109_closed_school.csv from the sample CSV folder
- Click the generate new preview button
- Download export CSV
- Confirm data is updated

## Screenshots
<img width="1139" alt="Screen Shot 2019-07-15 at 4 33 00 AM" src="https://user-images.githubusercontent.com/50601724/61203910-be194b00-a6b9-11e9-9ae2-7de537e78e68.png">

<img width="456" alt="Screen Shot 2019-07-15 at 4 34 32 AM" src="https://user-images.githubusercontent.com/50601724/61203971-e5701800-a6b9-11e9-9fdb-2a054394e9c5.png">

<img width="595" alt="Screen Shot 2019-07-15 at 4 35 29 AM" src="https://user-images.githubusercontent.com/50601724/61204005-059fd700-a6ba-11e9-9cf7-efa08bd8aa32.png">


## Acceptance criteria
- [x] The user can upload a CSV called “closed_school_sec_109” to the GIDS tool.
- [x] The “closure109” field in the Institutions.csv is populated using the new “closed_school_sec_109.csv” file.
- [x] The institutions.csv will match the “closed_school_sec_109.csv" based on Facility code and use that information to populate the column.
- [x] The column "closure109" data is always blank.

## Definition of done
